src/windows/include/display_device/windows/types.h

```diff
@@ -171,4 +172,18 @@ namespace display_device {
    * @brief Default function type used for cleanup/guard functions.
    */
   using DdGuardFn = std::function<void()>;
+
+  /**
+   * @brief Settings for workarounds/hacks for Windows.
+   */
+  struct WinWorkarounds {
+    /** @see win_utils::blankHdrStates for more details. */
+    std::optional<std::chrono::milliseconds> m_hdr_blank_delay { std::nullopt };
```

Comment:
```suggestion
  /**
   * @brief Settings for workarounds/hacks for Windows.
   * @seealso{win_utils::blankHdrStates for more details.}
   */
  struct WinWorkarounds {
    std::optional<std::chrono::milliseconds> m_hdr_blank_delay { std::nullopt };
```

Reference: https://docs.lizardbyte.dev/projects/libdisplaydevice/latest/md_third-party_2doxyconfig_2docs_2source__code.html#custom-aliases ... same for Sunshine.